Model/EraseEntityManagement.php

```diff
@@ -98,7 +98,7 @@ private function retrieveScheduledAt(): string
         );
     }
 
-    private function resolveErasureDelay(): int
+    public function resolveErasureDelay(): int
     {
         return (int)$this->scopeConfig->getValue(self::CONFIG_PATH_ERASURE_DELAY);
     }
```

Comment:
I think it would makes sense to move this method as a getter into `Opengento\Gdpr\Model\Config\Entity\Erasure`, and use this config provider in `Opengento\Gdpr\Model\EraseEntityManagement` and `Opengento\Gdpr\Model\Customer\Notifier\MailSender`

What do you think?

Reply:
Yes this is better, done in last commit